app/marian-decoder-new.cpp

```diff
@@ -34,10 +34,16 @@ int main(int argc, char *argv[]) {
   std::string input = std_input.str();
   using marian::bergamot::Response;
 
-  // Wait on future until Response is complete
-  std::future<Response> responseFuture = service.translate(std::move(input));
+  marian::bergamot::ResponseOptions responseOptions;
```

Comment:
`responseOptions` is not initialised but passed to `translate()`. Is that a better option that we use a default value for `responseOptions` in `translate()`? So we don't need `responseOptions` here.

Reply:
Removed `responseOptions`.